gneiss/composition/tests/test_composition.py

```diff
@@ -23,15 +23,19 @@ def test_ilr(self):
                              index=[1, 2, 3],
                              columns=['a', 'b', 'c', 'd'])
         table = table.reindex(columns=np.random.permutation(table.columns))
-        ph = pd.Series([1, 2, 3], index=table.index)
-        tree = gradient_linkage(table, ph)
+        # fix tree
+        tree = TreeNode.read([r"((d,c)y2, (b,a)y1)y0;"])
```

Comment:
Is this problem really just in the test suite? Seems like the underlying commands are producing different results, from what I can tell...

Reply:
found the problem - turns out that any operations you perform on a pandas column object directly impacts the pandas table itself.  I wonder if this is a bug in pandas - I'm going to open an issue.